src/django/pfb_analysis/models.py

```diff
@@ -184,33 +184,42 @@ def create_environment(**kwargs):
             logger.warn('Attempt to re-run job: {}. Skipping.'.format(self.uuid))
             return
 
-        client = boto3.client('batch')
-
-        # Since we run django manage commands in the analysis container, it needs a copy of
-        # all the environment variables that this app needs, most of which are conveniently
-        # prefixed with 'PFB_'
-        # Set these first so they can be overridden by job specific settings below
```

Comment:
Should we be clobbering these values with settings from `os.environ`? I would think we'd want it the other way around, where custom, explicitly set options overwrite whatever defaults we pull from the environment.